base/src/WebRTCSink.cpp

```diff
@@ -0,0 +1,95 @@
+#include <boost/filesystem.hpp>
+#include "WebRTCSink.h"
+#include "Command.h"
+#include "Frame.h"
+
+class WebRTCSink::Detail
+{
+public:
+    Detail(WebRTCSinkProps& _props) : mProps(_props)
+    { 
+    }
+
+    ~Detail()
+    { 
+    }
+
+    void setProps(WebRTCSinkProps _props)
+    {
+        mProps = _props;
+    }
+
+   
+public:
+    WebRTCSinkProps mProps;
+};
+
+
+
+WebRTCSink::WebRTCSink(WebRTCSinkProps _props)
+    :Module(SINK, "WebRTCSink", _props)
+{
+    mDetail.reset(new Detail(_props));
+}
+
+WebRTCSink::~WebRTCSink() {}
+
+bool WebRTCSink::validateInputPins()
+{   
+    return true;
+}
+
+bool WebRTCSink::validateOutputPins()
```

Comment:
sink - output pin validation does not make sense

Reply:
ValidateOutputPins is removed